modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Bare Metal Operator]
+ifdef::cluster-caps[= Bare metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Bare Metal Operator is an optional cluster capability that can be disabled by cluster administrators if it is not needed. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
+
+[discrete]
+== Purpose
+
+ifdef::cluster-caps[]
+
+The Bare Metal Operator provides the features for the `baremetal` capability. For more information, see _Cluster Operators reference_.
```

Comment:
it's a bit odd to refer people out to "cluster operators reference" for more information, when that page contains almost the exact same content as what is here (i.e. it doesn't really contain much in the way of additional info).

Also can we make these actual links?

Reply:
Unfortunately, our docs tooling on the Customer Portal keeps us from using cross references in modules. We can only add `xrefs` in assemblies. 

I will clean up these inconsistencies.